server/errors.go

```diff
@@ -94,6 +94,8 @@ func (*AggregatorServiceUnavailableError) Error() string {
 	return "Aggregator service is unreachable"
 }
 
+// DataEngineeringServiceUnavailableError error is used when
```

Comment:
oops?:) missing error type

Reply:
Oops! I realized that I don't need it but forgot to remove the comment :-)